### PR TITLE
YETUS-936 test-patch.sh does not do gradle

### PR DESCRIPTION
**Pull request by aw-was-here**

### CODE REVIEW

**Comment by apache-yetus** [comment]
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://builds.apache.org/job/yetus-github-multibranch/job/PR-88/1/console in case of problems.

**Comment by apache-yetus** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 46s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 14s |  Maven dependency ordering for branch  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  4s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  markdownlint  |   0m  2s |  There were no new markdownlint issues.  |
| +1 :green_heart: |  shellcheck  |   0m  7s |  There were no new shellcheck issues.  |
| +1 :green_heart: |  shelldocs  |   0m  1s |  There were no new shelldocs issues.  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 15s |  The patch does not generate ASF License warnings.  |
|  |   |   2m 48s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.6 Server=19.03.6 base: https://builds.apache.org/job/yetus-github-multibranch/job/PR-88/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/yetus/pull/88 |
| Optional Tests | dupname asflicense markdownlint shellcheck shelldocs |
| uname | Linux 41e05a05a5a2 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | precommit/src/main/shell/personality/yetus.sh |
| git revision | master / 5b1fb16 |
| Max. process+thread count | 50 (vs. ulimit of 2000) |
| modules | C: asf-site-src precommit U: . |
| Console output | https://builds.apache.org/job/yetus-github-multibranch/job/PR-88/1/console |
| versions | git=2.7.4 maven=3.3.9 shellcheck=0.7.0 markdownlint=0.15.0 |
| Powered by | Apache Yetus 0.12.0-SNAPSHOT https://yetus.apache.org |


This message was automatically generated.

File: precommit/src/main/shell/test-patch.d/checkstyle.sh [hunk 1]
```diff
@@ -202,10 +217,39 @@ function checkstyle_runner
...
+              else
+                newline=$(echo "$line" | "${SED}" -E 's,^.+line="([0-9]+)".*severity="([a-z]+)".*message="([^"]+)".+$,__FN__:\1:\3,')
+              fi
+              newline=$(echo "$newline" | "${SED}" -e "s,__FN__,$fn," -e "s,\&apos;,\',g" -e "s,\&quot;,\",g")
+              echo "$newline" >> "${tmp}"
+            fi
+          done < <(cat "${BASEDIR}/build/reports/checkstyle/main.xml" "${BASEDIR}/build/reports/checkstyle/test.xml")
+          mkdir -p "${PATCH_DIR}/${repostatus}-checkstyle"
+          cp  "${BASEDIR}/build/reports/checkstyle/main.xml" "${BASEDIR}/build/reports/checkstyle/test.xml" "${tmp}" \
+               "${PATCH_DIR}/${repostatus}-checkstyle"
```
Comment (aw-was-here) [inline precommit/src/main/shell/test-patch.d/checkstyle.sh]: Woops, these two lines were supposed to be removed.

**Comment by apache-yetus** [comment]
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://builds.apache.org/job/yetus-github-multibranch/job/PR-88/2/console in case of problems.

**Comment by apache-yetus** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 38s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 13s |  Maven dependency ordering for branch  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  4s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  markdownlint  |   0m  1s |  There were no new markdownlint issues.  |
| +1 :green_heart: |  shellcheck  |   0m  8s |  There were no new shellcheck issues.  |
| +1 :green_heart: |  shelldocs  |   0m  0s |  There were no new shelldocs issues.  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 15s |  The patch does not generate ASF License warnings.  |
|  |   |   2m 40s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.6 Server=19.03.6 base: https://builds.apache.org/job/yetus-github-multibranch/job/PR-88/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/yetus/pull/88 |
| Optional Tests | dupname asflicense markdownlint shellcheck shelldocs |
| uname | Linux 1469092846cb 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | precommit/src/main/shell/personality/yetus.sh |
| git revision | master / 301c6d6 |
| Max. process+thread count | 50 (vs. ulimit of 2000) |
| modules | C: asf-site-src precommit U: . |
| Console output | https://builds.apache.org/job/yetus-github-multibranch/job/PR-88/2/console |
| versions | git=2.7.4 maven=3.3.9 shellcheck=0.7.0 markdownlint=0.15.0 |
| Powered by | Apache Yetus 0.12.0-SNAPSHOT https://yetus.apache.org |


This message was automatically generated.

File: precommit/src/main/shell/test-patch.d/gradle.sh [hunk 2]
```diff
@@ -23,65 +23,47 @@ declare -a GRADLE_ARGS=()
...
 {
-  local i
-
-  for i in "$@"; do
-    case ${i} in
-      --gradle-cmd=*)
-        delete_parameter "${i}"
-        GRADLE=${i#*=}
-      ;;
-      --gradlew-cmd=*)
```
Comment (busbey) [inline precommit/src/main/shell/test-patch.d/gradle.sh]: usage above still claims we can pass a gradlew comand in. either remove it or add the handling back in here.

File: precommit/src/main/shell/test-patch.d/gradle.sh [hunk 3]
```diff
@@ -23,65 +23,47 @@ declare -a GRADLE_ARGS=()
...
-        delete_parameter "${i}"
-        GRADLEW=${i#*=}
-      ;;
-    esac
-  done
-
   # if we requested offline, pass that to mvn
```
Comment (busbey) [inline precommit/src/main/shell/test-patch.d/gradle.sh]: nit: comment should say pass to gradle

**Comment by busbey** [comment]
I'm +1 once the build tool usage message issue is addressed, so you can avoid blocking on me if you like.

File: precommit/src/main/shell/test-patch.d/gradle.sh (hunk 2 above)
Comment (aw-was-here) [reply precommit/src/main/shell/test-patch.d/gradle.sh]: Doh! Yeah that's clearly wrong. Thanks!

File: precommit/src/main/shell/test-patch.d/gradle.sh (hunk 3 above)
Comment (aw-was-here) [reply precommit/src/main/shell/test-patch.d/gradle.sh]: Yeah. c-n-p bug. haha.

**Comment by aw-was-here** [comment]
OK, I'm going to ahead and merge this.  Thanks @busbey !  Even if things are still a bit scewy (e.g., spotbugs) we're still in better shape than before.

**Comment by PierreSmits** [comment]
Looks good! Thanks.

**Pull request merged**